tigershark/facade/f835.py

```diff
@@ -472,3 +474,5 @@ def __init__(self, anX12Message):
                 self.loops(ClaimsOverview, anX12Message))
             self.claims = self.loops(Claim, anX12Message)
             self.footer = first(self.loops(Footer, anX12Message))
+            
+            self.loop = anX12Message
```

Comment:
why do need this line?

Reply:
it's __init__ line for X12LoopBridge class